raiden-dapp/src/services/config-provider.ts

```diff
@@ -35,4 +27,21 @@ export class ConfigProvider {
     }
     return undefined;
   }
+
+  static async configuration(): Promise<Configuration> {
+    const configurationUrl =
+      process.env.VUE_APP_CONFIGURATION_URL || '/config.json';
```

Comment:
These CONFIGURATION_URLs are all absolute. This is going to give issues with the relative deployment of `development` mode and `serve_pr` utilities. We'd better have relative URLs.

Reply:
Thanks, I will check and update accordingly.